python/fusion_engine_client/applications/p1_print.py

```diff
@@ -36,7 +38,10 @@ def add_print_format_argument(parser, *arg_names):
              "- oneline-binary-payload - Like `oneline-binary`, but exclude the message header from the binary")
 
 
-def print_message(header, contents, offset_bytes=None, format='pretty', bytes=None):
+def print_message(header, contents, offset_bytes=None, format='pretty', bytes=None, logger=None):
```

Comment:
Code organization wise, it seems like it would be a little clearer to break out these display functions into their own file. I also think it would be less confusing not to pass in a logger, but just use the normal logging level control mechanisms for this shared code. I assumed you just needed to do that since this logger had "application" in it's name?

Reply:
I can move these to a separate file. I passed the logger in since from the user's perspective these prints come from the application and it was odd to have it print from a different logging path. If I'm moving this to a separate file it's probably ok to remove this.